geometry_msgs/PointStamped safe_landing_spot
float32                    normal_z
